public/js/Login.js

```diff
@@ -38,11 +38,33 @@ function login() {
     const pathParts = window.location.pathname.split('/');
     const roomPath = pathParts[pathParts.length - 1];
 
-    if (username && password) {
+    let event = null;
+    let signedEvent = null;
+    const asyncKeyWrangler = async () => {
+        const publicKey = await window.nostr.getPublicKey();
+        event = {
+            kind: 27235,
+            pubkey: publicKey,
+            created_at: Math.floor(Date.now() / 1000),
+            content: "hivelogin",
```

Comment:
ref: https://github.com/nostr-protocol/nips/blob/master/98.md

content should be ""